formats/mdm/MDM_2_4.ddl

```diff
@@ -0,0 +1,515 @@
+-- MORA Data Message parser
+--
+-- This parser is implemented in accordance with the details by Chapter
+-- 6 ("MORA DATA MESSAGING") and related chapters of the Modular Open
+-- RF Architecture Draft Specification, version 2.4, published March 1,
+-- 2021. Where appropriate, the parser will make references to chapters
+-- and sections in the aforementioned document.
+
+import Daedalus
+
+def Main = MDM
+
+def MDM =
+  block
+    header = MDM_Header
+    body = MDM_Body header.type
+
+-- See: 6.1.2.1, MDM Header
+def MDM_Header =
+  block
+    -- See 6.1.2.1.1, MDM Preamble
+    -- "MDM"
+    Match [0x4d, 0x44, 0x4d]
```

Comment:
This could also be written as `Match "MDM"`

Reply:
Thank you, fixed in 7990c9deb57172fe3e0c7039a17f4824775ec9b9